Feature/slider blog update

robertswan opened the pull request.

robertswan committed:
fixed screen size on mobile. faq started

robertswan committed:
update to index, contact, and faq. more detail needed

robertswan committed:
faq updated. 1st pass complete

robertswan committed:
remove incomplete pages (devlog, & press)

robertswan committed:
corrections to faq platforms

robertswan committed:
fix pateron typo in faq

ExtraAzza committed:
[NEW FEATURES] Added Slider and Basic CMS Blog

I have added a basic slider which can be extended past 4 with ease,
needs styling and images adding but this is mainly all for testing

I have also added a basic cms blog system with adding/editing/deleting
functionailty. This is intended to make your life easier when posting
updates.

Also added a view page so people can view the entire update as the
homepage only shows the title and the date it was posted

Sweet! Thats all for now!

ExtraAzza committed:
Fixed Windows 64Bit Download Link not working

Fixed the Windows 64bit download link not working on the downloads page

SAFCxHarter commented:
This branch now includes the fix for the Windows 64bit download link not working Rob

ExtraAzza committed:
Fixed the rest of the download links

Didn't realise all the other download links didn't work too
They have now been fixed!

SAFCxHarter commented:
Fixed all the other download links Rob... didnt realise they didnt all work :/
